tests/ImageProcessorCore.Tests/Formats/ExifDecoderTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using ImageProcessorCore.Formats;
+using System.Linq;
+
+namespace ImageProcessor.Tests.Formats
+{
+    public class ExifDecoderTests
+    {
+
+        private TiffDecoderCore OpenJpegApp1File(string file)
```

Comment:
To reduce size, I extracted the app1 segment from the jpeg file. The TiffDecoder should be able to handle these without issue. Going forward, I should be able to add more app1 segments to do more testing.